index.d.ts

```diff
@@ -3168,9 +3081,6 @@ declare namespace Eris {
     removeMessageReaction(channelID: string, messageID: string, reaction: string, userID?: string): Promise<void>;
     removeMessageReactionEmoji(channelID: string, messageID: string, reaction: string): Promise<void>;
     removeMessageReactions(channelID: string, messageID: string): Promise<void>;
-    removeRelationship(userID: string): Promise<void>;
-    searchChannelMessages(channelID: string, query: SearchOptions): Promise<SearchResults>;
-    searchGuildMembers(guildID: string, query: string, limit?: number): Promise<Member[]>;
     searchGuildMessages(guildID: string, query: SearchOptions): Promise<SearchResults>;
```

Comment:
re-introduce searchGuildMembers back (searchGuildMessages is a user-only/undocumented endpoint)
```suggestion
    searchGuildMembers(guildID: string, query: string, limit?: number): Promise<Member[]>;
```

Reply:
Thanks, must have removed this by accident